api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -105,8 +105,7 @@
 	 * @should throw AmbiguousOrderException if disconnecting multiple active drug orders with the
 	 *         same drug
 	 */
-	@Authorized({ PrivilegeConstants.EDIT_ORDERS, PrivilegeConstants.ADD_ORDERS })
-	public Order saveOrder(Order order, OrderContext orderContext) throws APIException;
+	@Authorized({ PrivilegeConstants.EDIT_ORDERS, PrivilegeConstants.ADD_ORDERS }) Order saveOrder(Order order, OrderContext orderContext) throws APIException;
```

Comment:
Any reason this is not public anymore ?

Reply:
@dilanthas  i got to see that for the pr i made,there were some code formats,i am going to make a new pr for this and tag you,thanks